### ISSUE TITLE
I can see my support tier and content to 'unlock' in a next tier.

### ISSUE THREAD

**Issue by carwack** [posva/onlysponsors#9]
I would suggest an option to see what tier you are supporting a certain person. 

For example, I'm supporting Posva with the lowest tier. But didn't see anything about that on his profile.

And an indication that shows what tier I should be in to see certain posts would be nice.
Like it would say, get tier X to unlock this awesome post.

**Comment by posva** [comment]
Don't you get a link to sponsor for the amount needed when you are already sponsoring?

**Comment by carwack** [comment]
Yes, it does when I click through on a post. But I think it would be nice to have this directly on the profile page don't you think? 😁

And also the tier I'm supporting would be cool to find under 'Sponsered by:' or something on the profile page.

**Comment by posva** [comment]
I'm not sure I understand since each post can be locked to different tiers

**Comment by carwack** [comment]
Maybe my explanation isn't great.

So I tried to put something together (no design skills ofc). Hopefully, this is more clear.
<img width="1680" alt="Screenshot 2021-01-05 at 16 10 55" src="https://user-images.githubusercontent.com/16015740/103663519-78b5a100-4f71-11eb-8a47-4ec67fdfe3c3.png">

So each post is locked like this for example on the profile page

**Comment by posva** [comment]
Ah, I see, so this is related to https://github.com/posva/onlysponsors/issues/6
Having some indication of the tier you are sponsoring the person on would be nice! The information should be available

**Comment by carwack** [comment]
Yes it is indeed related to #6 👍 
And that's indeed what I meant! Would be awesome
